base/reflection.jl

```diff
@@ -281,13 +289,19 @@ end
 
 function return_types(f::Function, types::ANY=Tuple)
     types = to_tuple_type(types)
-    rt = []
+    rt = Any[]
     for x in _methods(f,types,-1)
+        # if the method is a @generated function and the argument types are not concrete,
+        # skip the method as the generated function cannot be instantiated
```

Comment:
Yes, but we are just skipping over these generated  function definitions (or erroring out currently).  Should be print them differently noting that they are staged   and not showing the AST in this case?